pkg/scheduler/api/helpers/helpers.go

```diff
@@ -56,14 +56,14 @@ func Max(l, r *api.Resource) *api.Resource {
 	res.ScalarResources = map[v1.ResourceName]float64{}
 	if l.ScalarResources != nil {
 		for lName, lQuant := range l.ScalarResources {
-			if lQuant > 0 {
+			if lQuant >= 0 {
```

Comment:
Please add ut.

Reply:
I already added it, please see the test case 9